docs/source/workflow_manager.rst

```diff
@@ -0,0 +1,308 @@
+what we will do for the Gauss 2d data source, as we are not
+connecting it to another data source.
+
+.. figure:: images/input_variables.png
+    :align: center
+
+The list of constants for the Gauss 2d data source are
+the positions, heights and widths of the two Gaussians
+(G1 and G2). The Gaussians are centred at (-1, 1) and (1, 1),
+with amplitudes of -2 and -1, respectively. That is, the
+first Gaussian is the global minimum whereas the second
+is a local minimum.
+
+Select an Optimizer
+^^^^^^^^^^^^^^^^^^^
+
+Selecting the ``MCO`` tree-item brings up two panels at the right:
+
+``Available MCO Factories``
+    A tree-list of all the available optimizers, arranged by the plugin that has contributed them. Note that not all of these will be multi-criterion optimizers.
+
+``Configuration Options/Description``
+    A description of the selected optimizer.
+
+Select an optimizer and press the ``Add New MCO`` button.
+
+.. figure:: images/mco_algo.png
+    :align: center
+
+The optimizer is added as a tree-item under ``MCO``. Selecting this
+item brings up a single panel to the right:
+
+``Item Details``
+    Certain parameters that control how the optimizer works.
+
+Select ``CMA`` for the ``algorithm`` and set 1000 for
+``Allowed number of objective calls``.
+
+
+Select the Parameters
+^^^^^^^^^^^^^^^^^^^^^
+
+Under the optimizer are two further tree-items for setting the parameters and KPIs.
+
+Selecting the ``Parameters`` tree-item brings up two panels at the right:
+
+``Available MCO Parameter Factories``
+    A tree-list of all the available parameters for the optimizer.
+
+``Description``
+    The description of the selected parameter.
+
+When we specify a "parameter" as well as selecting a data source input
+(that is not fed by the output of another data source) we must also
+tell the optimizer how to treat that input. Is the parameter:
+
+- fixed (i.e. a constant)?
+
+- continuous, with a lower and upper bound?
+
+- categorical, a member of an ordered or unordered set?
+
+Certain optimizers can only optimize certain parameter types. For
+instance, gradient-based optimizers can only handle continuous
+parameters, not categorical (which don't have a gradient). The
+Nevergrad optimizer can handle all types, but for now we will
+only use continuous ("Ranged").
+
+.. figure:: images/ranged_parameter.png
+    :align: center
+
+Select the ``Ranged`` item and press the ``New Parameter`` button. A new
+panel appears at the top-right. This will contain a tab for each
+parameter added. A ``Ranged`` parameter tab has the following fields:
+
+``Name``
+    A drop-down list of data source inputs. Select the input "x", the x coordinate.
+
+``Lower bound``
+    Set the lower bound to -5.
+
+``Upper bound``
+    Set the lower bound to 5.
+
+``Initial value``
+    Slide this to anything (it doesn't matter to the Nevergrad optimizer).
+
+``N samples``
+    This has no meaning and can be ignored.
```

Comment:
Maybe we can make an issue to replace the `RangedMCOParameter` class in the `NevergradMCOFactory` with a parameter class that doesn't contain these unnecessary traits.

Reply:
I think the purpose was for grid sampling - so yes we could put this in the MCO model somehow (specifying different N for different parameters might be a problem).